### ISSUE TITLE
failed to accept ACME authorization

### ISSUE THREAD

**Issue by magick93** [tnozicka/openshift-acme#76]
Hello

I recently re-installed the openshift-acme controller. 

Some routes were successfully secured, however others were not. 

There error I see in the controller log is:
```
failed to accept ACME authorization: failed to create watcher for Route myproject/myapp: the server rejected our request for an unknown reason (get routes.route.openshift.io)
```

Does anyone know how to further debug or understand this?

**Comment by tnozicka** [comment]
> I recently re-installed the openshift-acme controller.

Have you also updated all ClusterRoles/Roles?


> the server rejected our request for an unknown reason (get routes.route.openshift.io)

I am not sure what's the exact cause of this error just from that. Could you be on OpenShift < 3.6? (`oc version`)

https://github.com/tnozicka/openshift-acme/blob/1ff7aafad5785d605cd4391dada957b95af50f69/pkg/controllers/route/exposer.go#L298-L301

You could try watching route manually e.g. with `oc get route/myapp -n myproject -w`, if that work you could try it under the same service account as the controller uses. But I'd expect you get some Auth error in that case, not unknown.

**Comment by magick93** [comment]
Thanks @tnozicka 

Yes, I believe I have updated all roles. I reinstalled, following your instructions. And certainly some routes work as expected. And 2 dont. The only thing that these routes have in common is:

- the are both sub-domains of the same parent domain.
- the are both services that point to a Wildfly server. I doubt this is relevant as the routes/services have no knowledge of what is running in the pod.

I am running the following version:

```
oc version
oc v3.6.0+c4dd4cf
openshift v3.6.0+c4dd4cf
kubernetes v1.6.1+5115d708d7
```

I will try manually monitoring as you suggestion.
